src/thematic/1_water/sfp-105-nl_bemonsteren-peilbuis/06_materialen.Rmd

```diff
@@ -0,0 +1,75 @@
+# Benodigdheden
+
+## Apparatuur
+
+```{=html}
+<!--
+Opsomming van de apparatuur die nodig is en verwijzingen naar de sip's (Standard Instrument Procedure; <protocol-code>-YYYY.NN) indien die er zijn.
+Indien geen specifiek sip's voorhanden, illustreer de toegelaten apparatuur met foto's, geef het exacte type apparatuur mee, ...
+-->
+```
+-   Geschikt materiaal voor het nemen van een grondwatermonsters uit peilbuis
+
+    -   Peristaltische slangenpomp (Figuur \@ref(fig:Figuur1) A), Eijkelkamp, 12 vdc, regelbaar in twee richtingen tussen de 100 en 2.300 ml/min, met zuigcapaciteit tot 9,5 m diepte.
+        Deze pomp is voorzien van een batterij met levensduur van 4,5 uur en een pompslang in silicone.
+        De werking en het onderhoud van de peristaltische pomp is beschreven in SIP-xxx (nog niet afgewerkt).
+
+    -   Kogelklepmonsternemer, Eijkelkamp, 35mm, 90cm in teflon (Figuur \@ref(fig:Figuur1) B).
+
+> ![(#fig:Figuur1) Foto A = peristaltische pomp, B = Kogelklepmonsternemer](./media/image2.png){width="6.302777777777778in" height="2.611111111111111in"}
+
+-   Boormachine met peristaltische kop (Figuur \@ref(fig:Figuur2))
+
+> ![(#fig:Figuur2) Foto Boormachine met peristatltische kop](./media/image1.png){width="2.170223097112861in" height="2.8593755468066493in"}
+
+-   Veldset voor het bepalen van de zuurtegraad en de geleidbaarheid: het analytische laboratorium beschikt over verschillende veldsets (WTW multi, pH-EC) met de nodige elektroden welke kunnen gebruikt worden op het terrein voor deze bepalingen.
+    De werking, het gebruik, de kalibratie en het onderhoud van deze veldsets wordt beschreven in SIP-054 (kwaliteitshandboek analytisch labo).
```

Comment:
Hier verwijzen naar https://sites.google.com/inbo.be/veldprotocollen/standaard-instrument-procedures-sip?